app/code/Magento/ReviewGraphQl/Service/GetReviewAverageRatingService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Service;
+
+/**
+ * Class GetReviewAverageRatingService
+ */
+class GetReviewAverageRatingService
```

Comment:
Please, move it to the `Review/Model`. As we discussed above, we may consider it as a part of domain logic that can be reused not only within the scope of GraphQl modules.

Reply:
Moved ✅